src/controls/OrbitControls.ts

```diff
@@ -409,7 +409,7 @@ class OrbitControls extends EventDispatcher {
       scope.domElement.addEventListener('contextmenu', onContextMenu)
       scope.domElement.addEventListener('pointerdown', onPointerDown)
       scope.domElement.addEventListener('pointercancel', onPointerCancel)
-      scope.domElement.addEventListener('wheel', onMouseWheel)
+      scope.domElement.addEventListener('wheel', onMouseWheel, { passive: true })
```

Comment:
I believe we need this parameter in `dispose` as well.

Reply:
I don't think we should, since [EventTarget.removeEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener) signature doesn't have `passive` prop in its `options`